nixnet/database/_frame.py

```diff
@@ -553,27 +556,21 @@ def mux_subframes(self):
         return self._mux_subframes
 
     @property
-    def pdus(self):
-        # actually returns typing.Iterable[_pdu.Pdu], but avoiding a circular import
-        # type: () -> typing.Iterable[typing.Any]
+    def pdu_properties(self):
+        # type: () -> typing.Iterable[types.PduProperties]
         """list of :any:`Pdu`: Get or set a list that maps existing PDUs to a frame.
 
         A mapped PDU is transmitted inside the frame payload when the frame is transmitted.
         You can map one or more PDUs to a frame and one PDU to multiple frames.
 
         Mapping PDUs to a frame requires setting three frame properties.
```

Comment:
This isn't true anymore. Maybe something like:

```
Mapping PDUs to a frame requires modifying this list of :any:`PduProperties`,
each of which requires the following three properties:
```

Reply:
Yeahthat makes more sense. I was referring to the underlying driver calls here but that's not necessary at this spot.